proposals/0010-vk-buffer-ref.md

```diff
@@ -163,6 +163,16 @@ A vk::BufferPointer can otherwise be used whereever the HLSL spec does not other
 
 Applying HLSL semantic annotations to objects of type vk::BufferPointer is disallowed.
 
+### Buffer Pointers and Aliasing
+
+By default, buffer pointers are assumed to be restrict pointers as defined by the C99 standard.
+
+An attribute vk::aliased_pointer can be attached to a variable, function parameter or a block member of buffer pointer type. It is assumed that the pointee of an object with this attribute can overlap with the pointee of any other object with this attribute.
+
+### Buffer Pointers and Address Space
+
+All buffer pointers are presumed to point into the same address space. No new address space attributes are proposed.
```

Comment:
Which address space? The default address space in HLSL is thread-local memory, which I assume isn't where these are expected to refer. I assume these are host memory, so we should state that explicitly.

Reply:
Done.